scripts/build_renewable_profiles.py

```diff
@@ -538,7 +538,9 @@ def create_scaling_factor(
     # filter plants for hydro
     if snakemake.wildcards.technology.startswith("hydro"):
         country_shapes = gpd.read_file(paths.country_shapes)
-        hydrobasins = gpd.read_file(os.path.join(BASE_DIR, resource["hydrobasins"]))
+        hydrobasins_path = os.path.join(BASE_DIR, resource["hydrobasins"])
+        resource["hydrobasins"] = hydrobasins_path
```

Comment:
Just to be sure: a path to hydrobasins is used further when calculating the inflow [here](https://github.com/pypsa-meets-earth/pypsa-earth/blob/d727d39e95a04a1c1fe7d5935921ad060ad3fdfc/scripts/build_renewable_profiles.py#L594-L596), right?

Reply:
Thats right.